### DISCUSSION TITLE
`backdrop-blur-3xl` not working

### DISCUSSION THREAD

**Discussion by princejoogie** [tailwindlabs/tailwindcss#6608] [Help]
**What version of Tailwind CSS are you using?**
v3.0.5

**What build tool (or framework if it abstracts the build tool) are you using?**
Next.js 12.0.7

**What version of Node.js are you using?**
v14.17.6

**What browser are you using?**
Chrome and Firefox

**What operating system are you using?**
Windows

**Reproduction URL**
https://play.tailwindcss.com/jX1GgpQThm

**Describe your issue**
Trying to create a glass-morphism card effect with `backdrop-blur-3xl`. searched up online, tried to copy code line by line but it doest seem to work for me. reproduction is up above ^. Am i missing some styles to put to make the backdrop filter effect work?

### code:
```html
<section class="relative">
    <div class="absolute w-[95%] h-[95%] rotate-[-5deg] rounded-2xl origin-top-left bg-blue-500 -z-10"></div>

    <div class="w-full text-white p-8 bg-blue-500 shadow-inner backdrop-filter rounded-2xl bg-opacity-30 backdrop-blur-3xl">
      <div>
        <h4 class="text-sm text-gray-300">My balance</h4>
        <h3 class="text-2xl font-semibold">₱748,100</h3>
      </div>

      <div class="flex items-center mt-6">
        <div class="flex-1">
          <p class="text-sm text-gray-300">Spend</p>
          <h3 class="text-xl">-₱2800</h3>
        </div>

        <div class="flex-1">
          <p class="text-sm text-gray-300">Profit</p>
          <h3 class="text-xl">+₱4100</h3>
        </div>
      </div>
    </div>
  </section>
  ```

### desired output:
![image](https://user-images.githubusercontent.com/47204120/146631577-ab83c522-bc2c-48b1-a9aa-d5287b9519a1.png)

### actual output:
![image](https://user-images.githubusercontent.com/47204120/146631571-46934065-3acc-43f0-b1e4-20951900d60f.png)

**Comment by ashish1497** [comment]
Hi, did you solve this?

**Reply by wongjn** [reply]
Seems to be fixed in the reproduction URL so I assume it has been fixed in a more recent version of Tailwind.
